Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -87,6 +87,10 @@ YUI.add('ez-platformuiapp', function (Y) {
                 type: Y.eZ.DashboardView,
                 parent: 'loginFormView',
             },
+            dashboardBlockBasedView: {
```

Comment:
I think you could reuse app.views.dashboardView first because we still have a dashboard view and also because otherwise app.views.dashboardView becomes completely useless since it references Y.eZ.DashboardView which is undefined (ez-dashboardview is not loaded anymore).